applications/FluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -180,6 +182,21 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("Execute", &AccelerationLimitationUtilities::Execute)
         ;
 
+    py::class_<MassConservationUtility>(m,"MassConservationUtility")
+    .def(py::init < Model&, Parameters >())
+    .def(py::init< ModelPart&, Parameters >())
+    .def("Initialize", &MassConservationUtility::Initialize)
+    .def("ComputePositiveVolume", &MassConservationUtility::ComputePositiveVolume)
+    .def("ComputeNegativeVolume", &MassConservationUtility::ComputeNegativeVolume)
+    .def("ComputeFlowOverBoundary", &MassConservationUtility::ComputeFlowOverBoundary)
+    .def("ComputeBalancedVolume", &MassConservationUtility::ComputeBalancedVolume)
+    .def("ComputeDtForConvection", &MassConservationUtility::ComputeDtForConvection)
+    .def("ApplyLocalCorrection", &MassConservationUtility::ApplyLocalCorrection)
+    .def("ApplyGlobalCorrection", &MassConservationUtility::ApplyGlobalCorrection)
+    .def("ReCheckTheMassConservation", &MassConservationUtility::ReCheckTheMassConservation)
```

Comment:
May I ask why `ReCheck` and not `Check`?

Reply:
I guess becasue this is performed at the end of the process to "recheck" if the volume at the end is correct. I can change it